tests/jenkins/envutils.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Based on envutils.sh from blueconfigs repo
+
+# NOTE: This file shall be sourced so that important variables are avail to other scripts
+[ -n "$ENVUTILS_LOADED" ] && return || readonly ENVUTILS_LOADED=1
+_setbk=$-
+unset $(set +x; env | awk -F= '/^(PMI|SLURM)_/ {print $1}' | xargs)
+set +x
+Red='\033[31m'
+Green='\033[32m'
+Yellow='\033[33m'
+Blue='\033[34m'
+ColorReset='\033[0m'
+
+log() (
+    set +x
+    log_type=${2:-INFO}
+    echo -e "[${3:-$Blue} $log_type $ColorReset] $1"
+)
+
+log_ok() { log "$1\n" ${2:-OK} $Green; }
+
+log_warn() { log "$1" ${2:-WARNING} $Yellow; }
+
+log_error() { log "$1" ${2:-ERROR} $Red; }
+
+
+# On error abort with a meaningful msg
+trap 'log_error "Exit code: ${?}"' ERR
+
+
+if [[ -z "$ADDITIONAL_ENV_VARS" && -n "$GERRIT_CHANGE_COMMIT_MESSAGE" ]]; then
+    ADDITIONAL_ENV_VARS=$(set +x; echo "$GERRIT_CHANGE_COMMIT_MESSAGE" | sed -n "s/^ENV_VARS://p")
+fi
+log "Checking for override hooks (ADDITIONAL_ENV_VARS or gerrit 'ENV_VARS:')..."
+set -$_setbk
+[ "$ADDITIONAL_ENV_VARS" ] && eval $ADDITIONAL_ENV_VARS
+
+
+bb5_run() (
```

Comment:
this is not used?

Reply:
Yes, this can be removed. I forgot it. Thanks for pointing that out :+1: